stories/index.js

```diff
@@ -26,3 +27,13 @@ storiesOf('Button', module)
         ]
     })
     .add('default', () => <FormGroup controlId={'formid'} muted={'this should be muted'} label={'Ask question'} type={'email'} placeholder={'placeholder text...'} text="Hello World!" />);
+
+storiesOf('Image', module).addParameters({
+    backgrounds: [
+        { name: 'light', value: '#fffef0' },
+        { name: 'dark', value: '#1e3b47' },
+        { name: 'purple', value: '#7955ff', default: true }
+    ],
+}).add('default', () => (
+    <Image src="https://i.pinimg.com/originals/aa/49/54/aa4954451f16847cb657e68e5f46538a.jpg"/>
```

Comment:
Maybe add local image instead?

Reply:
Image component is already merged into master, its not made by me. Do you still want me to change the url of Image compoennt in all of my branches? im sure there can be an easier solution

Reply:
You should submit only one pr using local image for image component. And fix all your branches where you use images to use local image.